perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_login_namespace_bbmri_test_persistent_shadow.java

```diff
@@ -23,75 +12,41 @@
  */
 
 
-public class urn_perun_user_attribute_def_def_login_namespace_bbmri_test_persistent_shadow extends urn_perun_user_attribute_def_def_login_namespace {
+public class urn_perun_user_attribute_def_def_login_namespace_bbmri_test_persistent_shadow
```

Comment:
I think, that ``bbmri-test`` namespace is no longer required and module can be removed. I'll ask and let you know.

Reply:
I have it confirmed, module for ``bbmri-test`` can be removed from the code.

Reply:
I have removed the module.